conda_build/windows.py

```diff
@@ -187,7 +187,8 @@ def msvc_env_cmd(bits, config, override=None):
         if int(py_ver[0]) >= 3:
             if int(py_ver.split(".")[1]) < 5:
                 version = "10.0"
-            version = "17.0"
+            else:
```

Comment:
Good catch! Why don't we have a clause for Python 3.5 here? like we do in lines 308- below?

Reply:
Changed in [8052095](https://github.com/conda/conda-build/pull/5829/commits/8052095de79825bc891e3fec467f5f73a54e7069)